### PR TITLE
Pv terms soft launch

### PR DESCRIPTION
**Pull request by patrickvinograd**
Aligns the terms modal on the user profile page with the changes I made to the RequiredTermsAcceptance component - mainly using the user profile attributes to decide whether terms are accepted instead of making a separate API call. 

There is still some fine-tuning to do to the terms display, I will open a ticket to track that.

### CODE REVIEW

**Comment by patrickvinograd** [comment]
@bshyong Assuming no changes, should I merge this into the `acct` branch before merging that to kudos-launch?

**Comment by bshyong** [comment]
@patrickvinograd, yes probably better to merge to acct so that it is easy to keep track of changes - been using acct as the base branch for all FE account creation features

File: src/js/user-profile/components/AccountManagementSection.jsx [hunk 1]
```diff
@@ -21,6 +26,29 @@ class AccountManagementSection extends React.Component {
...
   }
 
+  acceptAndClose = (arg) => {
+    this.props.acceptTerms(arg);
+    this.closeModal();
+  }
+
+  renderModalContents() {
+    const { terms } = this.props;
+    const termsAccepted = this.props.profile.healthTermsCurrent;
```
Comment (bshyong) [inline src/js/user-profile/components/AccountManagementSection.jsx]: new attribute on the user profile coming provided from the API?

**Review by bshyong — APPROVED** [review]
LGTM

**Review by U-DON — APPROVED** [review]
LGTM.

**Comment by patrickvinograd** [comment]
Closing this in favor of merging to the `acct` branch; will then merge that into kudos-launch.

**Pull request closed without merging**